policies/antitrust-policy.md

```diff
@@ -0,0 +1,56 @@
+# Commonhaus Foundation Anti-Trust Policy
+
+## Purpose
+
+The Commonhaus Foundation (CF) requires that all members, activities, projects, committees, and working groups comply with applicable anti-trust laws.
+
+## Scope
+
+This policy applies to all participants in Commonhaus Foundation activities, including but not limited to general members, project representatives, councilors, and contributors to CF projects. It is designed to complement the [Conflict of Interest Policy](./conflict-of-interest.md) and other governance documents of the foundation.
+
+## Commitment to Compliance
+
+By participating in official or informal CF activities, participants commit to upholding this policy and promoting fair competition.
```

Comment:
"CF activities" is not clearly defined. Does this policy extend to the conduct of CF members and participants in every aspect of their lives?

Reply:
Would something like this work better?

```suggestion
By participating in any official or informal activities of the Commonhaus Foundation that are directly related to its governance, projects, or operations, participants commit to upholding this policy and promoting fair competition.
```